render-go/api.go

```diff
@@ -22,14 +22,15 @@ func main() {
 	}
 
 	// db
-	db, err := sql.Open("postgres", dbUrl)
+	dbpool, err := pgxpool.Connect(context.Background(), os.Getenv("DATABASE_URL"))
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer dbpool.Close()
```

Comment:
Did the lack of a previous `Close()` here cause the rise in active connections on Render?

<img width="1042" alt="Screen Shot 2022-04-30 at 3 45 11 PM" src="https://user-images.githubusercontent.com/198/166125046-f9710604-dbde-4b49-a868-8d66780615fa.png">